virt/post_install/index.adoc

```diff
@@ -0,0 +1,91 @@
+:_content-type: ASSEMBLY
+[id="post-install-configuration-overview"]
+= Post-installation configuration overview
```

Comment:
I fear that this will be hard to maintain. Maybe create a symbolic link to the assembly if you're planning to keep it the same? Or is this meant to be a template for creating a similar page of our own? (I see that it isn't in the topic map yet, so perhaps that's your plan.)

I'd request that you make this addition in its own PR, either way.